packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/controller.js

```diff
@@ -1,7 +1,20 @@
-import { DISCOVER_LINK } from '../constants';
+import { DISCOVER_LINK, TRACKING_CHAPTER_1, TRACKING_NAME } from '../constants';
+import { TRACKING_SUFFIX } from './constants';
 
 export default class OctaviaLoadBalancerPoolsCtrl {
-  constructor() {
+  /* @ngInject */
+  constructor($state, $window, atInternet) {
     this.link = DISCOVER_LINK;
+    this.$state = $state;
+    this.$window = $window;
+    this.atInternet = atInternet;
+  }
+
+  goToDiscoverOptions() {
+    this.atInternet.trackClick({
+      name: `${TRACKING_CHAPTER_1}::${TRACKING_NAME}::${TRACKING_SUFFIX}::goto-discover-options`,
+      type: 'action',
+    });
+    this.$window.open(this.link, '_blank');
   }
```

Comment:
There is no need to have this as the tracking can be handled on the html side with the attribute track-on, track-name abd track-type